WTWSMS/decisions/WtWSMS_realm_decisions.txt

```diff
@@ -341,15 +364,10 @@ decisions = {
 				has_landed_title = e_wre
 				has_landed_title = e_byzantium
 			}
-			OR = {
-				religion_group = christian
-				religion_group = pagan_group
-			}
 			age = 16
 			prisoner = no
 			NOT = { trait = incapable }
 			NOT = { has_global_flag = rome_restored }
-			has_romance_culture_trigger = yes
```

Comment:
"I'm sorry, Imperator Bubba, but you cannot unite the Eastern and Western Roman Empires, which you control, because your culture is different from your subjects"

That's ridiculous and unrealistic. No one on the earth would dare tell the Roman Emperor that he can't reunite the East and West that he controls. If I control both the Eastern and Western Roman Empires, I should be able to unite them. No exceptions.